demo/index.html

```diff
@@ -127,19 +127,19 @@
         <div id="green" class="avatar green" tabindex="0"></div>
         <div id="orange" class="avatar orange" tabindex="0"></div>
 
-        <paper-tooltip for="red" class="custom" position="left">
+        <paper-tooltip for="red" class="custom" position="left" animation-delay="0">
```

Comment:
Nope, just used it as an example (default is 500, these appear insta-fast with no delay). I don't have to use an example if you don't think it's useful :)